keras/engine/topology.py

```diff
@@ -806,33 +856,78 @@ def output_shape(self):
                             'ill-defined for the layer. ' +
                             'Use `get_output_shape_at(node_index)` instead.')
 
-    def add_updates(self, updates, inputs):
+    def add_loss(self, losses, inputs=None):
+        if losses is None:
+            return
+        # Update self.losses
+        losses = to_list(losses)
+        if not hasattr(self, 'losses'):
+            self.losses = []
+        try:
```

Comment:
I'm fairly sure property is just a wrong word in comment. See for example https://github.com/fchollet/keras/blob/master/keras/layers/core.py#L736